src/config/config.ts

```diff
@@ -85,6 +85,7 @@ export const appConfig: AppConfig = {
     Q9RrzMLTqxKnVVn2+hVpk4F/8tMsGCdd4s/AJqEQBy5lsq7ji1B63XYqi5fc1SnJ
     EQIDAQAB
     -----END PUBLIC KEY-----`,
+  gtmTrackingId: "GTM-RKWC",
```

Comment:
I don't think you need to have this id in the config file. It's a bit more work having to declare it here and importing it in each file where it's needed. If I understand correctly, there's only one instance of GTM so there is no need to have a "qa" and "prod" one, so, if it makes sense, put the id directly in the code snippets.